src/_bentoml_impl/server/app.py

```diff
@@ -8,6 +8,7 @@
 import typing as t
 from pathlib import Path
 
+import anyio
```

Comment:
Why do we decide to only limit to one thread per API function?

Reply:
answered offline